src/main/kotlin/org/openredstone/chad/Chad.kt

```diff
@@ -130,6 +130,7 @@ fun main(args: Array<String>) = runBlocking {
             })
             putAll(database.getCommands().mapValues { staticCommand(it.value) })
             put("help", helpCommand(this))
+            put("fractal", fractalCommand(authorizedRoles))
```

Comment:
This should be moved above the helpCommand line or it won't show up in the help message. (Preferably also above the putAll line, so that the custom commands are together, but that's only a cosmetic thing)

Reply:
i dont understand

Reply:
Actually it might not matter (the help text generation appears to be lazy) but basically I would still move this line so that it's before the `putAll` line:
```
            put("fractal", fractalCommand(authorizedRoles))
            putAll(database.getCommands().mapValues { staticCommand(it.value) })
            put("help", helpCommand(this))
```